generator-core/pom.xml

```diff
@@ -5,24 +5,21 @@
     <parent>
         <artifactId>spring-boot-cli</artifactId>
         <groupId>br.com.netodevel</groupId>
-        <version>0.0.1.BUILD-SNAPSHOT</version>
+        <version>0.0.2.BUILD-SNAPSHOT</version>
```

Comment:
Consider using a more stable version instead of a snapshot version to ensure reproducibility.